server/lib/notifications.js

```diff
@@ -340,7 +340,7 @@ async function notifyByEmail(activity) {
 
     case activityType.COLLECTIVE_APPROVED:
       // Funds MVP
-      if (get(activity, 'data.collective.settings.fund') === true) {
+      if (get(activity, 'data.collective.type') === 'FUND' || get(activity, 'data.collective.settings.fund') === true) {
```

Comment:
Do we still need to check the `data.collective.settings.fund` flag now that we have `collectiveType=FUND`?

Reply:
For a short time yes, but that should be ultimately removed (once we update frontend and migrate previous entries).